README.md

```diff
@@ -34,36 +34,28 @@ Types will need to be regenerated any time the graphql.schema is changed.
 yarn run codegen
 ```
 
-### 4. Build
-
-The project will need to be rebuilt any time any TypeScript source files are changed, including when types are regenerated.
-
-_NB: Also runs `codegen`._
-
-```shell
-# Builds for poktroll testnet.
-yarn run build
-
-# OR
-# Builds for poktroll localnet.
-yarn run build:develop
-```
-
-### 5. Run locally
+### 4. Run
 
 Build & start:
 ```shell
+# Run this only if indexing poktroll localnet. This will allows subquery-node to connect with the poktroll validator
+# leave this open in a separated terminal
+yarn docker:tunnel
```

Comment:
This will only work _after_ `yarn run docker:start` because the proxy service won't be listening until then. Let's **either**:

a. Move this step below the `yarn run docker:start` step.
b. #PUC to include this detail.

Reply:
This is working because if you check what that `yarn docker:tunnel` is doing on the update it also startup the proxy with `docker compose up -d proxy` so you can turn on this before anything else without issues and leave it running out there (if needed of course)